gem/app/views/surfnperf/_head.html.erb

```diff
@@ -7,6 +7,9 @@
   SURF_N_PERF.marks.pageStart = (new Date()).getTime();
 
   if(window.performance) {
+    window.performance.onresourcetimingbufferfull = function(){
+      SURF_N_PERF.resourcetimingbufferfull = true;
```

Comment:
I decided this functionality isn't that useful since you really need to listen for the vent to make use of it, so I'm removing this requirement. You can take out these 3 lines here